manifests/database.pp

```diff
@@ -0,0 +1,15 @@
+# sets up separate postgresql instance for pulpcore
+# @api private
+class pulpcore::database {
+
+  class { 'postgresql::globals':
+    version => '10',
+    client_package_name => 'rh-postgresql10-postgresql-syspaths',
+    server_package_name => 'rh-postgresql10-postgresql-server-syspaths',
+    contrib_package_name => 'rh-postgresql10-postgresql-contrib-syspaths',
+    service_name => 'postgresql_pulpcore',
```

Comment:
This implies you want to run a postgres instance just for pulp, but the module doesn't support that.